src/k8s-engine/hyperkit.ts

```diff
@@ -426,7 +433,8 @@ export default class HyperkitBackend extends events.EventEmitter implements K8s.
         resources.executable('docker-machine-driver-hyperkit'),
         ['--storage-path', path.join(paths.state(), 'driver'),
           'ssh', '--', 'sudo',
-          '/usr/local/bin/k3s', 'server'
+          '/usr/local/bin/k3s', 'server',
+          '--https-listen-port', this.cfg.port.toString()
```

Comment:
Maybe consider putting this in a local variable, in case `this.cfg` mutates between here and when we set `currentPort`?

Reply:
I don't see this -- if the desired port is changed after the server is launched, k8s will need to be recreated to work with the new port.

Reply:
Right, I'm more thinking of users who do things like start, and then change the port number _before_ the start is finished.  That will get our state to be out of whack (because `this.cfg.port` would be different between the start of the function and the end of it).

Reply:
I got why while working out the flow -- `start()` makes a lot of async calls, so it can process events while `await`ing those calls to finish. So while JS doesn't expose threads, it does still have race conditions.